_i18n/en/general/privacy.md

```diff
@@ -5,30 +5,30 @@ AntennaPod is developed and maintained by individual volunteers and is not repre
 ### Data AntennaPod may collect, store, and process
 The app only records data strictly necessary for the app to function. This includes the list of podcasts you follow, the app settings and authentication data provided for specific podcasts or services (such as synchronization services). The app does not include any advertising libraries or third-party tracking or analytics code, such as Google Analytics.
 
-**All data provided and created when using the AntennaPod app is stored locally on your device. AntennaPod does not send or upload your data anywhere, except where strictly necessary for the functioning of the app or when you actively share debugging information with us.** The next section explains when your data may reach third parties. If you're interested, you may also read about AntennaPod's [app permissions](/documentation/general/app-permissions) in the documentation.
+**All data provided and created when using the AntennaPod app is stored locally on your device. AntennaPod does not send or upload your data anywhere, except where strictly necessary for the functioning of the app or when you actively share debugging information with us.** The next section explains when your data may reach third parties. If you are interested, you may also read about AntennaPod's [app permissions](/documentation/general/app-permissions) in the documentation.
 
-When sharing debugging information (either via email using the ‘Crash Report’ function or via Android’s crash reporting functionality), the following data is processed:
+When sharing debugging information (either via email using the 'Crash Report' function or via Android's crash reporting functionality), the following data is processed:
```

Comment:
Could you replace the first two with the other ticks please? Like the other buttons: `